.pre-commit-config.yaml

```diff
@@ -0,0 +1,51 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.6.0
+    hooks:
+      - id: trailing-whitespace
+        # `.rs` files are taken care of by `cargo fmt`
+        exclude: ^.*\.rs$
+      - id: end-of-file-fixer
+      - id: check-merge-conflict
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    # A fast Python linter and code formatter. See
+    # https://docs.astral.sh/ruff/ for more details.
+    rev: v0.6.5
+    hooks:
+      - id: ruff
+        args: [--fix]
+        exclude: ^pineappl_cli/src/plot.py
+      - id: ruff-format
+        args: []
+        # TODO: remove this exclusion once we've merged this into master
```

Comment:
Please remember this :upside_down_face:

Reply:
You're right... This can be removed in master (I checked that the `plot.py` is not modified as it is already properly formatted).

Reply:
Done in commit cbe04a44fc3f0fcf87aa85ce3aec397eab9898ee.